4-storage/redshift-storage/sql/com.snowplowanalytics.snowplow/ua_parser_context_1.sql

```diff
@@ -0,0 +1,48 @@
+-- Copyright (c) 2014 Snowplow Analytics Ltd. All rights reserved.
+--
+-- This program is licensed to you under the Apache License Version 2.0,
+-- and you may not use this file except in compliance with the Apache License Version 2.0.
+-- You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+--
+-- Unless required by applicable law or agreed to in writing,
+-- software distributed under the Apache License Version 2.0 is distributed on an
+-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+--
+-- Authors:       Fred Blundun
+-- Copyright:     Copyright (c) 2015 Snowplow Analytics Ltd
+-- License:       Apache License Version 2.0
+--
+-- Compatibility: iglu:com.snowplowanalytics.snowplow/ua_parser_context/jsonschema/1-0-0
+
+CREATE TABLE atomic.com_snowplowanalytics_ua_parser_context_1 (
```

Comment:
@fblundun - table name should be com_snowplowanalytics_snowplow_ua_parser_context_1